### PR TITLE
GPG-603 Adds alternative reporting journey

### PR DESCRIPTION
**Pull request by dulciejackson**
Fixes some final stuff for the load test, including adding the alternative reporting journey so both versions of the reporting journey (swapped with a feature flag) can be tested.

### CODE REVIEW

File: Infrastructure/gpg-ipdeny/manifest.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 applications:
   - name: gpg-ipdeny
-    instances: 1
-    memory: 256M
+    instances: 3
+    memory: 3G
```
Comment (Binney) [inline Infrastructure/gpg-ipdeny/manifest.yml:6]: Is this a valid number? I'd have expected it to be 2 or 4

File: Infrastructure/gpg-ipdeny/manifest.yml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 applications:
   - name: gpg-ipdeny
-    instances: 1
-    memory: 256M
+    instances: 3
```
Comment (Binney) [inline Infrastructure/gpg-ipdeny/manifest.yml:5]: Is this on all environments? feels very high for dev, test and preprod - would've expected those to be 1 each. And we're going to autoscale live anyway right?

File: Infrastructure/gpg-ipdeny/nginx.conf [hunk 3]
```diff
@@ -4,7 +4,7 @@ daemon off;
 error_log /dev/stderr;
 
 events {
-  worker_connections 1024;
+  worker_connections 10000;
```
Comment (Binney) [inline Infrastructure/gpg-ipdeny/nginx.conf:7]: did this do anything in the end? if not I'd vote for keeping it as the default 1024

**Review by Binney — APPROVED** [review]

File: Infrastructure/gpg-ipdeny/manifest.yml (hunk 2 above)
Comment (dulciejackson) [reply Infrastructure/gpg-ipdeny/manifest.yml:5]: Ah yeah, good point. We haven't deployed the IP deny app to test or preprod yet so I hadn't given them much thought - I'll take this back down to 1 for now!

File: Infrastructure/gpg-ipdeny/nginx.conf (hunk 3 above)
Comment (dulciejackson) [reply Infrastructure/gpg-ipdeny/nginx.conf:7]: Yeah, it fixed my initial errors - now we're seeing timeouts and 500s but before I was seeing connections being dropped when it reached the 1024 threshold and couldn't take on more. 10000 was recommended on a StackOverflow post about configuring these values for live environments, which is why I went for it!

**Pull request merged**